src/Symfony/Component/Workflow/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
  * Added context to `TransitionException` and its child classes whenever they are thrown in `Workflow::apply()`
  * Added `Registry::has()` to check if a workflow exists
+ * Added support for `$context[Workflow::DISABLE_ANNONCE_EVENT] = true` when calling `workflow->apply()` to not fire the annonce event
```

Comment:
I think you have a type throughout the whole PR: `annonce -> announce`

Reply:
Thanks. Fixed